packages/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -6,25 +6,22 @@
       class="sf-heading--left sf-heading--no-underline title"
     />
     <div class="form">
-      <SfLoader :loading="loading">
-        <div v-if="error.loadMethods">
-          {{ $t('There was some error while trying to fetch shipping methods. We are sorry, please try with different shipping details or later.') }}
-        </div>
-        <div v-else-if="errorShippingProvider.save">
+      <SfLoader :loading="loadingShippingProvider">
+        <div v-if="errorShippingProvider.save">
```

Comment:
IMO it's better to place `errorShippingProvider.save` into a variable and maybe I'm wrong but if `errorShippingProvider` is computed you need to access the value in this way `errorShippingProvider.value.save`

Reply:
In the template we do not have to use .value